Kosystem/wwwroot/css/paper-nav-fixes.css

```diff
@@ -0,0 +1,65 @@
+﻿
+
+@media only screen and (max-width:768px) {
+    nav.no-collapse {
+        display: flex;
+    }
+}
+
+@media only screen and (max-width:768px) {
+    nav .incollapsible ul.inline li {
+        display: inline-block;
+        margin: 0 .5rem;
+    }
+}
+
+@media only screen and (max-width:992px) {
+    nav .incollapsible ul {
+        text-align: start;
+    }
+}
+
+nav .incollapsible input[id^=incollapsible]:checked ~ div.incollapsible-body {
+    margin: 0;
+    max-height: 960px;
+    opacity: 1;
+    padding: 0;
+}
+
+nav .incollapsible .incollapsible-body,
+nav .incollapsible:nth-of-type(1) {
+    border: 0;
+}
+
+/*@media only screen and (min-width:769px) {*/
+nav .incollapsible .incollapsible-body,
+nav .incollapsible:nth-of-type(1) {
+    display: contents;
+}
+/*}*/
+
+nav div.incollapsible-body {
+    padding: 0;
+}
+
+nav .incollapsible label {
+    border-bottom: 0;
+    padding: 0;
+}
+
+nav .incollapsible > button {
+    background-color: #fff;
+    display: none;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/HelloWorldSweden/Kosystem/pullRequest?prid=5636989)